### ISSUE TITLE
one-var not triggering a warning for mixed syntax

### ISSUE THREAD

**Issue by neemzy** [eslint/eslint#2570]
Hi,

I just noticed the `one-var` rule, when activated, does not trigger a warning for cases such as :

```javascript
var one = 1, two = 2;
var three;
```

If the rule is partially respected (first line), the violation on the second line isn't detected.
I'm using ESLint v0.21.1.

Thanks !

**Comment by ablin** [comment]
:+1:

**Comment by lo1tuma** [comment]
I can reproduce this with the [online demo](http://eslint.org/demo/).

```js
/*eslint one-var: [2, "always"]*/

var one = 1, two = 2;
var three;
```

Doesn’t give a warning, but

```js
/*eslint one-var: [2, "always"]*/

var one = 1, two = 2;
var three;
var four;
```

gives **one** warning for `var four;`.

**Comment by lo1tuma** [comment]
I think there is also a bug in an existing test-case: [`function foo() { var bar = true; var baz = false; var a; var b;}`](https://github.com/eslint/eslint/blob/40353732976da89e8902b6839e625f044ab4ed7d/tests/lib/rules/one-var.js#L319) with options `uninitialized: "always", initialized: "always"` is expected to only have 2 errors, but IMHO this should have 3 errors.
